encryption.rst

```diff
@@ -163,6 +239,65 @@ In this case it is necessary to use the ``--encrypt`` flag since the
 presence of an environment variable alone will not trigger the encrypted
 build workflow.
 
+.. code:: 
+
+   $ {command} sif list encrypted.sif
+
+Example:
+
+.. code:: 
+
+   ------------------------------------------------------------------------------
+   ID   |GROUP   |LINK    |SIF POSITION (start-end)  |TYPE
+   ------------------------------------------------------------------------------
+   1    |1       |NONE    |32176-32215               |Def.FILE
+   2    |1       |NONE    |32215-36269               |JSON.Generic
+   3    |1       |NONE    |36269-36465               |JSON.Generic
+   4    |1       |NONE    |36864-42954752            |FS (Encrypted squashfs/*System/arm64)
+   5    |1       |4       |42954752-42955497         |Cryptographic Message (PEM/RSA-OAEP)
+
+Note that ``ID=4`` partition has the type of ``Encrypted squashfs``
```

Comment:
```
Note that partition 4 is type ``Encrypted squashfs``.
```

Reply:
thanks for the comments
https://github.com/apptainer/apptainer-userdocs/pull/179/commits/816c972aee9ca359516da01997481793ca62c2ca